sources/packages/api/src/services/institution-location/institution-location.models.ts

```diff
@@ -21,3 +21,13 @@ export interface InstitutionLocationModel extends AddressInfo {
   primaryContactEmail: string;
   primaryContactPhone: string;
 }
+
+/**
+ * Service model for Institutions Institution location edit.
```

Comment:
From API perspective I would like describe it as Institution location update instead of edit.

Reply:
The comment must be improved.